eng/sendtohelix.proj

```diff
@@ -38,8 +38,8 @@
 
   <PropertyGroup Condition="'$(HelixType)' == ''">
     <!-- For PRs we want helixtype to be the same for all frameworks except package testing-->
-    <TestFilter Condition="'$(TestFilter)' == ''">innerloop</TestFilter>
-    <HelixType>test/functional/cli/$(TestFilter)/</HelixType>
+    <TestFilter Condition="'$(TestScope)' == ''">innerloop</TestFilter>
```

Comment:
Doesn’t TestFilter have to be TestScope? If empty is passed down as TestScope then you will assign innerloop to TestFilter but never use it and then HelixType will be `test/functional/cli//`

Reply:
lol of course, I made that change in the browser and missed that.